src/devtools/useAtomDevtools.ts

```diff
@@ -51,6 +67,18 @@ export function useAtomDevtools<Value>(
   }
 
   const [value, setValue] = useAtom(anAtom, scope)
+
+  const setAnyAtom = (value: Value) => {
+    if (isWritable(anAtom)) {
+      ;(setValue as SetAtom<Value, void>)(value)
```

Comment:
I guess you want to type guard `setValue`, not `anAtom`.
What's the type of `setValue`? Let me know if you need help.